tests/Integration/Http/Controller/ForgotControllerTest.php

```diff
@@ -118,35 +112,4 @@ public function resetPasswordNotFindingUserCorrectlyDisplaysMessage()
             $client->getResponse()->getContent()
         );
     }
-
-    /**
-     * @test
-     */
-    public function resetPasswordHandlesNotSendingResetEmailCorrectly()
```

Comment:
At the moment, I don't see a good way to simulate that Swiftmailer was unable to send mails. 😞